File: liczba palindromiczna.py
a=int(input("wprowadz liczbe "))
b=a
odwrotnosc = 0
while (a>1):
    odwrotnosc+=a%10
    if (a>10):
        odwrotnosc=odwrotnosc * 10
    a=a//10
print("Liczba ",end="")
if (odwrotnosc != b): print("nie ",end="")
print("jest palindromiczna")

File: liczba palindromiczna.cpp
#include <iostream>
using namespace std;

int funkcja(int i)
{
	int g=i;
	int odwrocona=0;
	while (i>1)
	{
		odwrocona+=i%10;
		if (i>10) odwrocona*=10;
		i/=10;
	}
	if (odwrocona != g)
	{
		cout<<"nie ";
	}
	cout<<" jest palindromiczna";
	return 0;
}

int main()
{
	int i;
	cin>>i;
	int palindromiczna = funkcja(i);
}
